docs/agent-install-standalone-linux.md

```diff
@@ -0,0 +1,115 @@
+# Install to Linux using a GZIP File
+
+Install the Smart Agent to Linux host using a compressed
+standalone package in a GZIP file.
+
+## Prerequisites
+
+* Kernel version 2.6.32 or higher
+* cap_dac_read_search and cap_sys_ptrace capabilities
+* Terminal or a similar command-line interface application
+* GZIP application
+
+## Install the Smart Agent using a GZIP file
+
+1. Remove collector services such as `collectd`
+
+2. Remove third-party instrumentation and agent software
+
+> Don't use automatic instrumentation or instrumentation agents from
+> other vendors when you're using SignalFx instrumentation. The results
+> are unpredictable, and your instrumentation might break and your
```

Comment:
I think this only applies to APM autoinstrumentation, not the agent.  Any "unpredictability" in the agent due to other agents would be a bug in the agent.

Reply:
I removed this note and the bullet point that precedes it. It was actually in all of the files. Don't know why.